examples/scala/src/main/scala/example/Streaming.scala

```diff
@@ -70,6 +74,8 @@ object Streaming {
     stream2.awaitTermination(10000)
     stream2.stop()
 
+
+    println("=== Section 3: Streaming upgrades in update mode")
```

Comment:
I'd phrase this as "Streaming upserts using MERGE".

Reply:
Yes, I updated with your comment.